### DISCUSSION TITLE
为什么会有get访问其他地址？

### DISCUSSION THREAD

**Discussion by reedlxm** [illacloud/illa-builder#411] [Q&A]
我在我的测试环境搭建了illa builder，docker服务的ip地址是40，ip地址是41的服务器上部署了一台Django测试服务器。
当我登录illa并进入app编辑界面的时候。看到旁边Django服务器的状态开始刷屏。显示不断有人访问GET /room/CLOUD/dashboard
有点好奇，是bug么？

**Comment by karminski** [comment]
感谢反馈, 我们检查一下相关的接口路由. 最近的确有改动这一部分的设计. 其中 /room/CLOUD/dashboard 是websocket 服务的访问 path. 需要配置一个叫 WEBSOCKET_SERVER_ADDRESS 的环境变量. 目前不确定之不是有这部分引起的. 
一旦有新的进展, 我们会立刻回复您.

**Comment by karminski** [comment]
@reedlxm 我们更新了构建方式，在服务器上部署可以试试这个：

https://github.com/illacloud/deploy-illa-manually

期待您的反馈
